### PR TITLE
Proposal for SIG Benchmarking

### PR DESCRIPTION
**Pull request by cartersocha**
Updated the charter and SIG proposal per GC/TC feedback

### CODE REVIEW

File: projects/perf-benchmark.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+performance and customer resources seriously. Performance tracking and
+quantification is a general need that should be addressed by a project wide
+effort and automated tooling that minimizes repo owner effort while providing
+valuable new data points for all project stakeholders.  
+
+### Project Board
+
+### SIG Charter
+
+[charter](https://docs.google.com/document/d/1W0hLqHrUNtS2yclORTNtFZ-lXDblMzUYrp1PCSyzJr8/edit?usp=sharing)
```
Comment (arminru) [inline projects/perf-benchmark.md:25]: Once finalized, adding it to the markdown here would be good so it's preserved in the repository.

File: projects/perf-benchmark.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  current performance numbers and track changes as new features / PRs are
+  merged.
+* Basic implementation of the existing specification and repo specific desired
+  tests in the Collector and 3 language repos.
+* Provide ongoing maintenance as needed on automated tooling and own the
+  underlying assets
+
+Initial implementation scope would be the core Collector components (main repo),
+JavaScript / Java / Python SDKs and their core components. No contrib or
+instrumentation.
```
Comment (arminru) [inline projects/perf-benchmark.md:41]:
```suggestion
instrumentation repositories.
```

File: projects/perf-benchmark.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Anyone with an opinion on performance standards and testing.
+
+Language contributors as the SIG will be tasked with implementing the changes
+and following through on the process. Maintainers or approvers can optionally
+participate or consult the SIG on how to use the provided tooling.
+
+#### Required staffing
+
+lead - tbd @jpkrohling domain expert
```
Comment (arminru) [inline projects/perf-benchmark.md:53]: cc @jpkrohling

File: projects/perf-benchmark.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+@cartersocha contributor
+@mwear collector sig
+@codeboten collector sig implementation
```
Comment (arminru) [inline projects/perf-benchmark.md:56]: What's the difference? 🙂

File: projects/perf-benchmark.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+Need: more performance domain experts, language implementers
+
+### Meeting Times
+
+TBD
+
+### Timeline
+
+Initial scope is for the Collector and 3 SDKs. Output should be by KubeCon NA
+November 6, 2023
```
Comment (arminru) [inline projects/perf-benchmark.md:74]:
```suggestion
Initial scope is for the Collector and 3 SDKs. Initial output should be available by KubeCon NA
November 6, 2023.
```
Is there a session where the results are intended to be presented?

Comment (cartersocha) [reply projects/perf-benchmark.md]: Not a specific session  but was hoping to mention it in general project updates

File: projects/perf-benchmark.md (hunk 3 above)
Comment (cartersocha) [reply projects/perf-benchmark.md:53]: this was a formatting issue. The lead is tbd but will update the doc

File: projects/perf-benchmark.md (hunk 4 above)
Comment (cartersocha) [reply projects/perf-benchmark.md]: Removed implementation

File: projects/perf-benchmark.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+Anyone with an opinion on performance standards and testing.
+
+Language contributors as the SIG will be tasked with implementing the changes
+and following through on the process. Maintainers or approvers can optionally
+participate or consult the SIG on how to use the provided tooling.
+
+#### Required staffing
+
+lead - tbd
```
Comment (jpkrohling) [inline projects/perf-benchmark.md:53]: @cartersocha, any luck finding a lead for this?

**Comment by jpkrohling** [comment]
After talking to @cartersocha on Slack, I'm closing this. If there's renewed interest in this, let us know and we can reconsider.

**Pull request closed without merging**